vercel.json

```diff
@@ -0,0 +1,5 @@
+{
+  "github": {
+    "silent": true
```

Comment:
What does this do?

Reply:
Removed, thanks—only relevant to the setup where Vercel does the docs build on each PR and push to main. (It stops the Vercel bot from commenting on every PR)